assets/scripts/opt-out.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+(function ($) {
+  // Triggers out-out cookie creation when click on link.
+  $('.core-standards-optout-link').on('click', function(e) {
```

Comment:
The missing space after the keyword `function` should have triggered an eslint compile error – please double-check with @LucaPipolo whether the frontend build pipeline is correctly set up for the core-standards plugin.

Reply:
@sun As far as I see, there is no build system set for this plugin (no package.json, no gulpfile.json...).